package.json

```diff
@@ -10,12 +10,20 @@
     "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
   },
   "devDependencies": {
```

Comment:
Upgraded all of these in my desperate attempt to figure out why the components weren't loading, only to discover that nothing worked until I deleted my node modules and re-installed and then everything worked perfectly. 🙄